ooniapi/services/oonifindings/src/oonifindings/routers/v1.py

```diff
@@ -122,7 +134,7 @@ class OONIFindingIncidents(BaseModel):
 @router.get(
     "/v1/incidents/search",
```

Comment:
I think there is an opportunity here to improve a bit the semantics of these endpoints. We should speak to @majakomel about it though.

For example if we want to make this a truly restful endpoint, then we should not have the verb in the endpoint, but rather just have it be `/v2/findings?[query_params]`